tools/unitctl/Cargo.lock

```diff
@@ -676,11 +870,49 @@ checksum = "0fafdf7b2b2de7c9784f76e02c0935e65a8117ec3b768644379983ab333ac98c"
 dependencies = [
  "futures-util",
  "hex",
- "hyper",
+ "hyper 0.14.27",
  "pin-project",
  "tokio",
 ]
 
+[[package]]
+name = "hyperlocal-next"
+version = "0.9.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "acf569d43fa9848e510358c07b80f4adf34084ddc28c6a4a651ee8474c070dcc"
+dependencies = [
+ "hex",
+ "http-body-util",
+ "hyper 1.3.1",
+ "hyper-util",
+ "pin-project-lite",
+ "tokio",
+ "tower-service",
+]
+
+[[package]]
+name = "iana-time-zone"
+version = "0.1.60"
```

Comment:
Heh, the more TZ data the better!

Actually, looks like this is the culprit for bringing in the Android stuff...

Reply:
Not sure what I can do about this specific import at the moment.

Reply:
Sure, it was more a tongue a cheek comment about everyman and his dog bringing their own copy of the timezone database.

Then I noticed the android dependencies of which they also bring the timezone database (unless that's the only reason of the dependency!)